src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -335,6 +335,7 @@ private static MemoryInfoResult GetMemoryInfoLinux(SmapsReader smapsReader)
                     commitedMemoryInBytes = cgroupMemoryUsage.Value;
                     fromProcMemInfo.Commited.Set(commitedMemoryInBytes, SizeUnit.Bytes);
                     fromProcMemInfo.MemAvailable.Set(maxMemoryUsage - cgroupMemoryUsage.Value, SizeUnit.Bytes);
+                    fromProcMemInfo.AvailableWithoutTotalCleanMemory.Set(maxMemoryUsage - cgroupMemoryUsage.Value + fromProcMemInfo.SharedCleanMemory.GetValue(SizeUnit.Bytes), SizeUnit.Bytes);
```

Comment:
Can this ever be negative?

Reply:
How? ..
It takes `maxMemoryUsage` (which is the containers max mem), decrement container's usage which always less then max usage, and adding zero or above clean mem.
Unless cgroup usage is extremely high (and there's an if condition for this block to prevent that and decide we are inside a container), then no negative number can be produced..

Reply:
User define a limit that is higher than the mem? 
OS weirdness? 
Even if negative, would this break stuff?

Reply:
@aviviadi ?

Reply:
done